src/Queries/Dates/February.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Having to split these up is a pretty gigantic blowup. Each of these were five lines before, now they are a separate file and 14 lines. Is there an option to have multiple of these classes in the same file without violating the PSR standards?